src/components/products/list/ProductsList.js

```diff
@@ -83,24 +99,50 @@ class ProductsList extends Component {
 						>
 							Add Product
 						</Button>
-					</Col>
-				</Row>
 
-				<Row>
-					<Col>
-						<div className="products-list-wrapper">
-							<SimpleList
-								elementsList={prods}
-								titleFieldName="name"
-								subtitleFieldName="code"
-								deletable
-								editable
-								clickable
-							/>
-						</div>
+						<ButtonGroup className="buttons-layout-change float-right">
+							<Button variant="secondary" onClick={this.renderListView}>
+								<FontAwesomeIcon icon={faList} size="2x" />
+							</Button>
+							<Button variant="secondary" onClick={this.renderGridView}>
+								<FontAwesomeIcon icon={faTh} size="2x" />
+							</Button>
+						</ButtonGroup>
 					</Col>
 				</Row>
 
+				{layoutState ? (
+					<Row>
+						<Col>
+							<div className="products-list-wrapper">
+								<SimpleList
+									elementsList={prods}
+									titleFieldName="name"
+									subtitleFieldName="code"
+									deletable
+									editable
+									clickable
+								/>
+							</div>
+						</Col>
+					</Row>
+				) : (
+					<Row className="product-tile">
```

Comment:
you either have to remove Row from here or from ProductTile because what happens now is <row><row><youTileStuff /></row></row>